doc/dev_guide/deply_noobaa_on_minikube_or_rancher_desktop.md

```
@@ -130,6 +138,19 @@ Once noobaa is installed please notice that the phase is Ready, you will see it
+NAME                                               READY   STATUS    RESTARTS        AGE
+noobaa-operator-ccb78496b-kf4rs                    1/1     Running   0               5m
+cnpg-controller-manager-69b76d949c-w4c8n           1/1     Running   0               5m
+noobaa-db-pg-cluster-1                             1/1     Running   0               3m36s
+noobaa-db-pg-cluster-2                             1/1     Running   0               2m55s
+noobaa-core-0                                      2/2     Running   0               2m43s
+noobaa-endpoint-577466dd97-6r9sd                   1/1     Running   0               95s
+noobaa-default-backing-store-noobaa-pod-6a58d8de   1/1     Running   0               73s
+```
+
+* Note if you would run the install with the flag `--use-standalone-db` then you can see something similar to this when getting the pods (single db pod instead of 3):
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)